### PR TITLE
Implement delete Internship feature (deletei command)

### PR DESCRIPTION
**Pull request by ish1506**
Some auto-formatting fixes were done by Intellij.

### CODE REVIEW

File: src/main/java/seedu/address/logic/Logic.java [hunk 1]
```diff
@@ -17,10 +17,11 @@
 public interface Logic {
     /**
      * Executes the command and returns the result.
+     *
      * @param commandText The command as entered by the user.
      * @return the result of the command execution.
      * @throws CommandException If an error occurs during command execution.
-     * @throws ParseException If an error occurs during parsing.
+     * @throws ParseException   If an error occurs during parsing.
```
Comment (LiuFangrui) [inline src/main/java/seedu/address/logic/Logic.java:24]: empty space

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public class EditCommand extends Command {
     private final EditPersonDescriptor editPersonDescriptor;
 
     /**
-     * @param index of the person in the filtered person list to edit
+     * @param index                of the person in the filtered person list to edit
```
Comment (LiuFangrui) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:58]: empty space

File: src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java [hunk 3]
```diff
@@ -7,11 +7,11 @@
 
 /**
  * Tokenizes arguments string of the form: {@code preamble <prefix>value <prefix>value ...}<br>
- *     e.g. {@code some preamble text t/ 11.00 t/12.00 k/ m/ July}  where prefixes are {@code t/ k/ m/}.<br>
+ * e.g. {@code some preamble text t/ 11.00 t/12.00 k/ m/ July}  where prefixes are {@code t/ k/ m/}.<br>
  * 1. An argument's value can be an empty string e.g. the value of {@code k/} in the above example.<br>
  * 2. Leading and trailing whitespaces of an argument value will be discarded.<br>
  * 3. An argument may be repeated and all its values will be accumulated e.g. the value of {@code t/}
- *    in the above example.<br>
+ * in the above example.<br>
```
Comment (LiuFangrui) [inline src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java:14]: should be indented

**Comment by ish1506** [comment]
@LiuFangrui Fixed formatting issues alr!

**Pull request merged**